src/components/app/Home.js

```diff
@@ -25,29 +25,24 @@ import {
 } from 'firefox-profiler/app-logic/web-channel';
 import { assertExhaustiveCheck } from 'firefox-profiler/utils/flow';
 
-import type {
-  ConnectedProps,
-  WrapFunctionInDispatch,
```

Comment:
I spent too much time looking at this type `WrapFunctionInDispatch` and why it wasn't used in HomeImpl, and concluded that it _should_ be done automatically by `explicitConnect` but it's not since we upgraded to some version of Flow where we couldn't make it work.

Anyway let's not bother about that in this patch.